mode: 1
reinitialize: false
initial_with_ground_truth: true
plot_data: true

kalman_filter:
  forward_prediction_time: 0
  init_sigma_location: 0
  init_sigma_orientation: 0

  enable_prediction: true
  enable_data_association: true
  enable_correction: true
  

# EKF arguments
# cmd:=cmd_vel scan:=base_scan _plot_data:=true _mode:=1 _initial_with_ground_truth:=true _plot_data:=false _map_image:="/home/mbader/projects/catkin/mr2015ws/src/mr_mr2015/mr_self_localization/maps/cave.png" _map_lines:="/home/mbader/projects/catkin/mr2015ws/src/mr_mr2015/mr_self_localization/maps/cave.yml"